Formula/gambit-scheme.rb

```diff
@@ -27,27 +26,25 @@ class GambitScheme < Formula
   def install
     args = %W[
       --prefix=#{prefix}
+      --infodir=#{info}
       --enable-single-host
-      --enable-multiple-versions
       --enable-default-runtime-options=f8,-8,t8
       --enable-openssl
     ]
 
     system "./configure", *args
 
-    # Fixed in gambit HEAD, but they haven't cut a release
-    inreplace "config.status" do |s|
-      s.gsub! %r{/usr/local/opt/openssl(?!@1\.1)}, "/usr/local/opt/openssl@1.1"
-    end
-    system "./config.status"
-
     system "make"
     ENV.deparallelize
     system "make", "install"
+
+    # fix lisp file install location
+    mkdir_p share/"emacs/site-lisp/#{name}"
+    mv share/"emacs/site-lisp/gambit.el", share/"emacs/site-lisp/#{name}"
```

Comment:
```suggestion
    elisp.install share/"emacs/site-lisp/gambit.el"
```
Isn't this the same?